storage/replica.go

```diff
@@ -1151,6 +1156,97 @@ func (r *Replica) maybeSetCorrupt(err error) error {
 	return err
 }
 
+// resolveIntents resolves the given intents. Those which are local to the range
```

Comment:
Why is this broken out? This new method seems more complicated than what was previously in `Store.resolveWriteIntentError()`. We're unlikely to need a slow and fast path bifurcation from write intents encountered executing individual commands.